package.json

```diff
@@ -34,6 +34,7 @@
     "@types/dagre": "^0.7.46",
     "@types/graphlib": "^2.1.8",
     "babel-jest": "^29.5.0",
+    "copy-webpack-plugin": "^11.0.0",
```

Comment:
Why do we need this?

Reply:
Because when we are running the webpack to recompile, I need to use 
`
  plugins: [
    new CopyPlugin({
      patterns: [
        {from: './extraSettingPage/options.html', to: 'options.html'},
        {from: './extraSettingPage/options.js', to: 'options.js'},
      ],
    }),
  ],
`
to copy options.html and options.js to the final compiled folder.